src/Symfony/Component/HttpFoundation/Tests/RedirectResponseTest.php

```diff
@@ -44,6 +44,14 @@ public function testGenerateLocationHeader()
         $this->assertEquals('foo.bar', $response->headers->get('Location'));
     }
 
+    public function testGenerateContentTypeHeader()
+    {
+        $response = new RedirectResponse('foo.bar');
+
+        $this->assertTrue($response->headers->has('Content-Type'));
```

Comment:
This assertion can be removed as it is redundant with the next one.

Reply:
Done! Thank you